src/Microsoft.DocAsCode.Build.Engine/HostServiceConstructor.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.IO;
+    using System.Linq;
+
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    internal class HostServiceConstructor : IHostServiceConstructor
```

Comment:
ha, any difference?

---
In reply to: [89049826](https://github.com/dotnet/docfx/pull/913#discussion_r89049826) [](ancestors = 89049826)